src/main/java/org/mitre/synthea/world/geography/Demographics.java

```diff
@@ -39,7 +41,49 @@
   private RandomCollection<String> incomeDistribution;
   public Map<String, Double> education;
   private RandomCollection<String> educationDistribution;
+  
+
+  public static List<LinkedHashMap<String, String>> getSpewPeople() throws IOException {
+    List<LinkedHashMap<String, String>> spewPerson = SimpleCSV
```

Comment:
If these are going to be called for each person, we don't want to be reading and parsing the CSV each time, that would be really slow. It would be better to store them in a constant so that they can be re-used many times. Take a look at LifecycleModule.growthChart for an example. (though the name is actually a bad example, it's a constant so it should be ALL_UPPER_CASE)